kotlin-js-store/wasm/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
https://kotlinlang.org/docs/whatsnew22.html#build-infrastructure-for-wasm-target-separated-from-javascript-target